src/testdir/test_termdebug.vim

```diff
@@ -18,9 +18,8 @@ if g:GCC->empty()
   throw 'Skipped: gcc is not found in $PATH'
 endif
 
-packadd termdebug
-
-func Test_termdebug_basic()
+function s:generate_files(bin_name)
+  let src_name = a:bin_name . '.c'
   let lines =<< trim END
```

Comment:
The single dot for string concatenation is deprecated now.  Can you use ".." here?